tsconfig.json

```diff
@@ -1,6 +1,13 @@
 {
+  "compilerOptions": {
+    "jsx": "react-jsx"
+  },
   "extends": "@mate-academy/students-ts-config",
   "include": [
     "src"
+  ],
+  "lib": [
+    "ES6",
+    "DOM"
```

Comment:
We also have dom in libs, and we use `esnext` alias for using the last ES version